arq/worker.py

```diff
@@ -192,6 +197,7 @@ def __init__(
         on_job_end: Optional['StartupShutdown'] = None,
         after_job_end: Optional['StartupShutdown'] = None,
         handle_signals: bool = True,
+        job_completion_wait: int = 0,
```

Comment:
I see you've used `'SecondsTimedelta'` other places, but this was not accepted by mypy and `asyncio.wait_for()`.

Reply:
I think this is clear.